components/automate-ui/src/app/modules/infra-proxy/policy-groups-list/policy-groups-list.component.html

```diff
@@ -1,12 +1,12 @@
 <chef-table>
-  <chef-thead *ngIf="policyGroups.length">
+  <chef-thead *ngIf="policyFiles.length">
     <chef-tr class="no_border_tr">
       <chef-th class="no_border_th">Policy Group</chef-th>
       <chef-th class="no_border_th">Number of Policyfiles</chef-th>
     </chef-tr>
   </chef-thead>
   <chef-tbody>
-    <chef-tr *ngFor="let policyGroup of policyGroups">
+    <chef-tr *ngFor="let policyGroup of policyFiles">
```

Comment:
Names are not matching we have policygroups here why change?

Reply:
updated changes